payjoin-cli/contrib/lint.sh

```diff
@@ -6,5 +6,5 @@ features=("v1" "v2")
 
 for feature in "${features[@]}"; do
   # Don't duplicate --all-targets clippy. Clilppy end-user code, not tests.
-  cargo clippy --no-default-features --features "$feature" -- -D warnings
+  cargo clippy --no-default-features --features "$feature,_manual-tls" -- -D warnings
```

Comment:
There is a holdout with the `certificate_key` in the config being dead code when building with just v1 or v2 enabled.